modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -12,28 +12,36 @@ namespace access to the {CNVProductName} Operators.
 
 .Prerequisites
 
-* `OperatorGroup` and `CatalogSource` Custom Resource Definition objects (CRDs),
-both created in the `openshift-cnv` namespace
+* Create a namespace called `openshift-cnv`.
 
 .Procedure
 
-. Open a browser window and navigate to the {product-title} web console.
+. Open a browser window and log in to the {product-title} web console.
 
-. Navigate in the web console to the *Operators → OperatorHub* page.
+. Navigate to the *Operators → OperatorHub* page.
 
-. Scroll or type a keyword into the *Filter by keyword* box to find the KubeVirt
-HyperConverged Cluster Operator, then click on it.
+. Scroll or type a keyword into the *Filter by keyword* box to find the
+{CNVProductName} Operator, then click on it.
```

Comment:
s/{CNVProductName}/KubeVirt HyperConverged Cluster
why it was replaces  wise verse at first place?
![image](https://user-images.githubusercontent.com/6319198/66104645-8c799700-e5b9-11e9-9cff-825976cd7e54.png)

Reply:
Good question. I could have sworn that it was the Container-native Virtualization Operator in the material @rthallisey provided, but looking at it now I just see Kubevirt HyperConverged Cluster Operator. Will fix.